Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -127,6 +134,13 @@
     Telecrystal: 100 # novelty item for the larp, doesnt come with bundle though because bulky fuck
   categories:
   - UplinkWearables
+  conditions:
+  - !type:StoreWhitelistCondition
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)